tests/utils/esx/govc.go

```diff
@@ -59,21 +59,32 @@ func GetVMPowerState(vmName string) string {
 // PowerOnVM util powers on the VM
 // govc vm.power -on=true photon
 func PowerOnVM(vmName string) string {
-	// log.Printf("Powering on VM [%s]\n", vmName)
+	log.Printf("Powering on VM [%s]\n", vmName)
 	cmd := esx.PowerOnVM + vmName
 	out, _ := ssh.InvokeCommandLocally(cmd)
 	return out
 }
 
 // PowerOffVM util powers off the VM
+// Caution: If a VM is writing to disk when it receives a
+// Power Off command, data corruption may occur.
 // govc vm.power -off=true photon
 func PowerOffVM(vmName string) string {
-	// log.Printf("Powering off VM [%s]\n", vmName)
+	log.Printf("Powering off VM [%s]\n", vmName)
 	cmd := esx.PowerOffVM + vmName
 	out, _ := ssh.InvokeCommandLocally(cmd)
 	return out
 }
 
+// ShutDownVM util shuts down the VM
+// govc vm.power -s=true photon
+func ShutDownVM(vmName string) string {
+	log.Printf("Shuting down VM [%s]\n", vmName)
+	cmd := esx.ShutDownVM + vmName
```

Comment:
I see your point. So you are asking why we are not using govmomi API/library directly. @shuklanirdesh82 did the early investigation. My guess is that that govc CLI seems easier to integrate with. Let me file a issue to track this so that the current fix will not be blocked.